src/index.js

```diff
@@ -0,0 +1,26 @@
+const url = "https://api.github.com/search/users?q=";
+const userMode = "%20in:login";
+let item = document.querySelector(".items");
+const button = document.querySelector(".search_button");
+button.addEventListener("click", displayUser)
+
+async function getResource() {
```

Comment:
very common function name... if you are searching for a users than your function should be called `searchUsers` and it also has to have an argument - what you are looking for. so e.g. `searchUsers(username)` - and url inside this function should be dynamic :) otherwise you constantly will have same result